app/controllers/v0/documents_controller.rb

```diff
@@ -6,7 +6,13 @@ class DocumentsController < ApplicationController
     def create
       params.require :file
       claim = DisabilityClaim.for_user(current_user).find(params[:disability_claim_id])
-      jid = claim_service.upload_document(claim, params[:file], params[:tracked_item])
+      document_data = EVSS::DocumentData.new
+      document_data.evss_claim_id = claim.evss_id
+      document_data.file_name = params[:file].original_filename
+      document_data.tracked_item_id = params[:tracked_item]
+      document_data.document_type = params[:document_type]
```

Comment:
Not necessary in this PR but it would be nice to clean this up and use strong parameters, then just pass the whitelisted hash of args to a constructor instead of setting each attribute separately